dandi/files/zarr.py

```diff
@@ -209,32 +216,44 @@
         import zarr
 
         errors: list[ValidationResult] = []
+        origin: Origin = Origin(
+            validator=Validator.zarr,
+            validator_version=zarr.__version__,
+            standard=Standard.ZARR,
+            # TODO: standard_version=...,
```

Comment:
Per https://zarr.readthedocs.io/en/stable/user-guide/v3_migration.html#migration-guide, it seems that the Python `zarr` pacakge can support more than one Zarr format 3 specification at the same time. What is the way to determine which Zarr format specification is being used in just opening a file? My bet is that possibly all support format specifications are used, so the `standard_version` field may be irrelevant and should be skipped in this case.